tests/builder/ignore_errors.rs

```diff
@@ -124,3 +126,24 @@ fn subcommand() {
         Some("some other val")
     );
 }
+
+#[test]
+fn help_command() {
+    static HELP: &str = "\
+Usage: test
+
+Options:
+  -h, --help  Print help
+";
+
+    let cmd = Command::new("test").ignore_errors(true);
+
+    utils::assert_output(cmd, "test --help", HELP, false);
+}
+
+#[test]
+fn version_command() {
+    let cmd = Command::new("test").ignore_errors(true).version("0.1");
+
+    utils::assert_output(cmd, "test --version", "test 0.1\n", false);
+}
```

Comment:
Could you split this PR into two commits?
- The first commit adds the tests where they pass with the existing behavior
- The second commit modifies the behavior and updates the tests so they are passing with the new behavior

This helps us
- Verify the tests are testing the correct behavior
- Help clarify to a reviewer what behavior is changing with a PR